src/taco-cli/cli/kit.ts

```diff
@@ -95,8 +104,9 @@ class Kit extends commands.TacoCommandBase {
         return Kit.promptUser(resources.getString("CommandKitSelectProjectUpdatePrompt"))
         .then(function (answer: string): Q.Promise<any> {
             if (answer && answer.length > 0 ) {
-                answer = answer.toLowerCase();
-                if (resources.getString("PromptResponseYes").split("\n").indexOf(answer) !== -1) {
+                // Try the lower-case version of the answer as well (This is needed so we can have better test coverage for "yes" and "no" response code paths)
```

Comment:
In the test mode, we do not resolve the actual strings from resources.json (resouces.getString("PromptResponseYes") will just return "PromptResponseYes"  which means that in order to test the positive code paths, we will have to pass "PromptResponseYes" (for this check to succeed). The goal was to get as much test coverage as possible.